### PR TITLE
add link to CLA

### PR DESCRIPTION
**Pull request by jburel**
see https://github.com/ome/omero-documentation/issues/2329

### CODE REVIEW

File: omero/developers/index.rst [hunk 1]
```diff
@@ -30,6 +30,7 @@ from |Glencoe| OMERO is released under the `GNU General Public License (GPL)`_
 with commercial licenses and customization available from |Glencoe|. You can
 read about how OMERO has developed since the project started in the
 :doc:`/users/history`.
+We ask contributors to sign a `Contributor License Agreement (CLA) <https://drive.google.com/file/d/1ld6s678JWRF15qUT4_3LyTE4IGYh-wRZ/view?usp=sharing >`_. The agreement only needs to be sign once.
```
Comment (dominikl) [inline omero/developers/index.rst]: Maybe add an email address where to send the complete CLA too. In our docs contributors@openmicroscopy.org is mentioned for that, but I don't know where this goes, I don't get emails sent to this address.

**Review by dominikl — APPROVED** [review]
Thanks @jburel , looks good 👍

**Comment by jburel** [comment]
@dominikl I had to make a few changes since your last view. Still okay?

**Comment by dominikl** [comment]
👍 yes, looks good.

**Pull request merged**